src/components/Chart/DonutChartStory.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { DonutChart } from './index';
+
+const patternfly = window.patternfly;
+
+/**
+ * DonutChart constants
+ */
+
+const donutConfigData = {
+  columns: [['Used', 60], ['Available', 40]],
+  groups: [['used', 'available']],
+  order: null,
+};
+const donutConfigTooltip = {
+  contents: patternfly.pfGetUtilizationDonutTooltipContentsFn('MHz'),
+};
+const donutConfigTitle = { primary: '60', secondary: 'MHz Used' };
+const donutData = {
+  colors: {
+    Cats: patternfly.pfPaletteColors.blue,
+    Hamsters: patternfly.pfPaletteColors.green,
+    Fish: patternfly.pfPaletteColors.orange,
+    Dogs: patternfly.pfPaletteColors.red,
+  },
+  columns: [['Dogs', 2], ['Cats', 2], ['Fish', 3], ['Hamsters', 1]],
+};
+
+const donutRightConfigData = donutData;
+const donutRightConfigLegend = {
+  show: true,
+  position: 'right',
+};
+const donutRightConfigTooltip = {
+  show: true,
+  contents: patternfly.pfDonutTooltipContents,
+};
+const donutRightConfigTitle = { primary: '8', secondary: 'Animals' };
```

Comment:
maybe this should have a function to auto calculate this for donut, something like https://github.com/theforeman/foreman/blob/1.16-stable/webpack/assets/javascripts/services/ChartService.js#L59

Reply:
👍 Added a `title.type` option:
when 'percent' do as forman do :-) , when 'total' do as patternfly example do, when we have explicit title use it.